R/plot-population-time-profile.R

```diff
@@ -0,0 +1,132 @@
+#' Concentration time profile plot for population
+#'
+#' @inheritParams plotIndividualTimeProfile
+#' @param quantiles A numerical vector with quantile values (Default: `c(0.05,
```

Comment:
"... with the quantile values defining the aggregation of individual data..."

Reply:
Done.